packages/core/admin/admin/src/components/CommandPalette/Container.js

```diff
@@ -0,0 +1,119 @@
+import styled from 'styled-components';
+
+export default styled.div`
+  z-index: 1001;
+  position: relative;
+  opacity: ${(props) => (props.open ? 1 : 0)};
+
+  [cmdk-overlay] {
+    background: ${({ theme }) => `${theme.colors.neutral800}1F`};
+
+    position: fixed;
+    inset: 0;
+  }
+
+  [cmdk-dialog] {
```

Comment:
Do you have to style the pieces this way? Can we not do `styled(Component)`?

Reply:
I can't render those as custom components from what I read in the docs and examples I saw but if you have some options I'm listening

Reply:
I'm not sure I see where you are going with this 🤔 do you have an example of what you would do ?

Reply:
Sure so just taking a reduced version of your `CommandPalette` component:

```jsx
	return (
   <CommandContext.Provider value={context}>
      <Container open={open}>
        <Command.Dialog
          open={open}
          onOpenChange={setOpen}
          label="Global Command Menu"
          container={containerElement.current}
        >
          <Command.Input
            autoFocus
            placeholder="What are you looking for?"
            onValueChange={setInputValue}
            value={inputValue}
            onKeyDown={onInputKeyDown}
          />
          <Command.List>
            <Command.Empty>No results found.</Command.Empty>
          </Command.List>
        </Command.Dialog>
      </Container>
    </CommandContext.Provider>
	)
```

We can do:

```jsx
const Dialog = styled(Command.Dialog)`
	  position: fixed;
    top: 20%;
    left: 50%;
    border-radius: 12px;
    max-width: 640px;
    width: 100%;
    overflow: hidden;
    box-shadow: 0 16px 70px rgba(0, 0, 0, 0.2);
    padding: 8px;
    transform-origin: 50%;
    transform: translateX(-50%) scale(1);
    outline: 0;
    background: ${({ theme }) => theme.colors.neutral100};
    border: 1px solid ${({ theme }) => theme.colors.neutral150};
`

	return (
   <CommandContext.Provider value={context}>
      <Container open={open}>
        <Dialog
          open={open}
          onOpenChange={setOpen}
          label="Global Command Menu"
          container={containerElement.current}
        >
          <Command.Input
            autoFocus
            placeholder="What are you looking for?"
            onValueChange={setInputValue}
            value={inputValue}
            onKeyDown={onInputKeyDown}
          />
          <Command.List>
            <Command.Empty>No results found.</Command.Empty>
          </Command.List>
        </Dialog>
      </Container>
    </CommandContext.Provider>
	)
```

etc. Does that make more sense? Wrapping `Command.Dialog` in `styled` passes a `className` to the component which means it'll work correctly and we're splitting up the CSS into clearer spaces

Reply:
Ok so that's something I already tried when implementing it but I couldn't make it work the command palette just wouldn't open anymore. I'll try it again now that it's almost ready that migh have been failing because of sth else